### PR TITLE
[POT] Update GNA HW config, aggregators, unify rules

### PR DESCRIPTION
**Pull request by KodiaqQ**
### Details:
 - *Added Concat unification for GNA*
 - *Updated Concat unification rule for DepthWise Convolution*
 - *Added **power_of_2** aggregator*

### CODE REVIEW

File: tools/pot/openvino/tools/pot/algorithms/quantization/fake_quantize_configuration.py [hunk 1]
```diff
@@ -328,7 +328,8 @@ def _is_concat_unify_condition(node):
         def _is_followed_by_conv(input_node):
             if _is_quantize_agnostic_op(input_node):
                 concat_stack.extend(get_all_node_outputs(input_node))
-            elif input_node.type in [n['type'] for n in CONCAT_UNIFY_OUTPUTS]:
+            elif input_node.type in [n['type'] for n in CONCAT_UNIFY_OUTPUTS] \
+                and not is_depth_wise(input_node):
```
Comment (AlexKoff88) [inline tools/pot/openvino/tools/pot/algorithms/quantization/fake_quantize_configuration.py:332]: Do you think that GNA is able to execute quantized DW Conv in a per-channel fashion? I think it should be verified.

Comment (KodiaqQ) [reply tools/pot/openvino/tools/pot/algorithms/quantization/fake_quantize_configuration.py:332]: Not sure, thanks for the comment. It was a general fix, that causes failure for ANY device quantization.
Do you think that DW Conv's exists at the GNA-like networks?

Comment (AlexKoff88) [reply tools/pot/openvino/tools/pot/algorithms/quantization/fake_quantize_configuration.py:332]: @nkogteva, can you please comment here? I remember you had some concerns about DW Conv quantization.

Comment (nkogteva) [reply tools/pot/openvino/tools/pot/algorithms/quantization/fake_quantize_configuration.py:332]: @AlexKoff88 We have a transformation that converts a DW separable convolution to a set of MatMul + Add. And we agreed at the meeting that this transformation should be moved to offline (POT). I created ticket 69672 for that.

Comment (AlexKoff88) [reply tools/pot/openvino/tools/pot/algorithms/quantization/fake_quantize_configuration.py:332]: Ok. But let's handle this separately. Is this really a set of MatMul +Add?

**Comment by ilya-lavrenov** [comment]
@KodiaqQ @nkogteva @AlexKoff88 is it still valid?

**Comment by AlexKoff88** [comment]
@KodiaqQ, I think we can close as it is not actual anymore. @KodiaqQ, please confirm.

**Pull request closed without merging**